try {
  fn1();
}
catch (err) {
    console.log("Kluda izpildot fn2.js:" +err);
    }